### PR TITLE
2.11.5 Release Notes

### PR DESCRIPTION
**Pull request by desertaxle**
[RENDERED](https://github.com/PrefectHQ/prefect/blob/d703bffced4d0d00ee639ac2c79a1a6bb8f69d8f/RELEASE-NOTES.md)

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+- Clarify deployments variables — https://github.com/PrefectHQ/prefect/pull/10474
+- Update README header image — https://github.com/PrefectHQ/prefect/pull/10493
+
+## Contributors
+- @mattklein made their first contribution in https://github.com/PrefectHQ/prefect/pull/10422
+- @vishalsanfran made their first contribution in https://github.com/PrefectHQ/prefect/pull/10394
+- @AmanSal1
+- @mj0nez
+
+**All changes**: https://github.com/PrefectHQ/prefect/compare/2.11.4...preview
```
Comment (serinamarie) [inline RELEASE-NOTES.md:43]:
```suggestion
**All changes**: https://github.com/PrefectHQ/prefect/compare/2.11.4...2.11.5
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+- Remove references to `flow_name` from deployments documentation — https://github.com/PrefectHQ/prefect/pull/10477
+- Update kubernetes guide for readability — https://github.com/PrefectHQ/prefect/pull/10481
+- Fix a few typos in docs — https://github.com/PrefectHQ/prefect/pull/10488
+- Surface more documentation on push work pools — https://github.com/PrefectHQ/prefect/pull/10497
+- Fix heading size, remove unnecessary link in deployments documentation — https://github.com/PrefectHQ/prefect/pull/10489
+- Create GCP specific guide references for deploying a GKE cluster to host a worker — https://github.com/PrefectHQ/prefect/pull/10490
+- fix typo in gcs recipe — https://github.com/PrefectHQ/prefect/pull/10442
+- Move and reference Agents -> Workers upgrade guide — https://github.com/PrefectHQ/prefect/pull/10445
+- docs: fixed a header and some wordings — https://github.com/PrefectHQ/prefect/pull/10457
+- Clarify deployments variables — https://github.com/PrefectHQ/prefect/pull/10474
```
Comment (serinamarie) [inline RELEASE-NOTES.md:34]:
```suggestion
- Clarify deployments variables and fix `prefect.yaml` example — https://github.com/PrefectHQ/prefect/pull/10474
```
might be a better way to say this

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+
+### Enhancements
+- Warn users with misconfigured `PREFECT_API_URL` — https://github.com/PrefectHQ/prefect/pull/10450
+- Show cli message for the paused work-pool  — https://github.com/PrefectHQ/prefect/pull/10369
+- Optimize the query generated by the /task_runs endpoint — https://github.com/PrefectHQ/prefect/pull/10422
+- add discord notification block — https://github.com/PrefectHQ/prefect/pull/10394
+- update wizard recommendation when no `pull` step selected — https://github.com/PrefectHQ/prefect/pull/10473
+
+### Fixes
+- Update base job template validation to only run on work pool update and creation — https://github.com/PrefectHQ/prefect/pull/10486
```
Comment (serinamarie) [inline RELEASE-NOTES.md:13]:
```suggestion
- Run base job template validation when work pools are updated or created — https://github.com/PrefectHQ/prefect/pull/10486
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,47 @@
...
+- Adding some safety checks to a prior optimization on task runs — https://github.com/PrefectHQ/prefect/pull/10466
```
Comment (serinamarie) [inline RELEASE-NOTES.md:14]:
```suggestion
- Extend optimization on `/task_runs` endpoint to include safety guards — https://github.com/PrefectHQ/prefect/pull/10466
```

Comment (serinamarie) [reply RELEASE-NOTES.md:14]: I see this as an enhancement

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,47 @@
 # Prefect Release Notes
 
+## Release 2.11.5
+
+### Enhancements
+- Warn users with misconfigured `PREFECT_API_URL` — https://github.com/PrefectHQ/prefect/pull/10450
+- Show cli message for the paused work-pool  — https://github.com/PrefectHQ/prefect/pull/10369
```
Comment (serinamarie) [inline RELEASE-NOTES.md:7]:
```suggestion
- Show CLI warning if worker is polling a paused work pool or queue  — https://github.com/PrefectHQ/prefect/pull/10369
```

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,47 @@
 # Prefect Release Notes
 
+## Release 2.11.5
+
+### Enhancements
+- Warn users upon setting a misconfigured `PREFECT_API_URL` — https://github.com/PrefectHQ/prefect/pull/10450
+- Show cli message for the paused work-pool  — https://github.com/PrefectHQ/prefect/pull/10369
+- Optimize the query generated by the `/task_runs` endpoint — https://github.com/PrefectHQ/prefect/pull/10422
+- add discord notification block — https://github.com/PrefectHQ/prefect/pull/10394
```
Comment (serinamarie) [inline RELEASE-NOTES.md:9]:
```suggestion
- Add `DiscordWebhook` notification block — https://github.com/PrefectHQ/prefect/pull/10394
```

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,47 @@
...
+- update wizard recommendation when no `pull` step selected — https://github.com/PrefectHQ/prefect/pull/10473
```
Comment (serinamarie) [inline RELEASE-NOTES.md:10]:
```suggestion
- Remove reference to deprecated `prefect project ls` in interactive `prefect deploy` command — https://github.com/PrefectHQ/prefect/pull/10473
```

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,47 @@
...
+- Adding some safety checks to a prior optimization on task runs — https://github.com/PrefectHQ/prefect/pull/10466
+
+### Experimental
+- Codify concurrency context managers and rate limiting with tests — https://github.com/PrefectHQ/prefect/pull/10414
+
+### Documentation
+- Guide to Kubernetes — https://github.com/PrefectHQ/prefect/pull/10368
+- Update warning block to mention worker in title — https://github.com/PrefectHQ/prefect/pull/10464
+- Fix duplicate Amplitude deviceId — https://github.com/PrefectHQ/prefect/pull/10449
+- Combine K8s worker page with K8s flows page — https://github.com/PrefectHQ/prefect/pull/10448
```
Comment (serinamarie) [inline RELEASE-NOTES.md:23]:
```suggestion
- Combine Kubernetes worker and flows pages — https://github.com/PrefectHQ/prefect/pull/10448
```

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,47 @@
...
+- Guide - How to move data to and from cloud providers — https://github.com/PrefectHQ/prefect/pull/10133
```
Comment (serinamarie) [inline RELEASE-NOTES.md:24]: I think this one could also move to the highlight with the other guide

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -1,5 +1,47 @@
...
+- Remove references to `flow_name` from deployments documentation — https://github.com/PrefectHQ/prefect/pull/10477
+- Update kubernetes guide for readability — https://github.com/PrefectHQ/prefect/pull/10481
```
Comment (serinamarie) [inline RELEASE-NOTES.md:26]:
```suggestion
- Improve readability of Kubernetes guide  — https://github.com/PrefectHQ/prefect/pull/10481
```

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -1,5 +1,47 @@
...
+- Fix a few typos in docs — https://github.com/PrefectHQ/prefect/pull/10488
+- Surface more documentation on push work pools — https://github.com/PrefectHQ/prefect/pull/10497
```
Comment (serinamarie) [inline RELEASE-NOTES.md:28]:
```suggestion
- Raise visibility of push work pools documentation — https://github.com/PrefectHQ/prefect/pull/10497
```

File: RELEASE-NOTES.md [hunk 12]
```diff
@@ -1,5 +1,47 @@
...
+- Fix heading size, remove unnecessary link in deployments documentation — https://github.com/PrefectHQ/prefect/pull/10489
+- Create GCP specific guide references for deploying a GKE cluster to host a worker — https://github.com/PrefectHQ/prefect/pull/10490
+- fix typo in gcs recipe — https://github.com/PrefectHQ/prefect/pull/10442
+- Move and reference Agents -> Workers upgrade guide — https://github.com/PrefectHQ/prefect/pull/10445
```
Comment (serinamarie) [inline RELEASE-NOTES.md:32]:
```suggestion
- Move guide on upgrading from agents to workers — https://github.com/PrefectHQ/prefect/pull/10445
```

File: RELEASE-NOTES.md [hunk 13]
```diff
@@ -1,5 +1,47 @@
...
+- Fix duplicate Amplitude deviceId — https://github.com/PrefectHQ/prefect/pull/10449
+- Combine K8s worker page with K8s flows page — https://github.com/PrefectHQ/prefect/pull/10448
+- Guide - How to move data to and from cloud providers — https://github.com/PrefectHQ/prefect/pull/10133
+- Remove references to `flow_name` from deployments documentation — https://github.com/PrefectHQ/prefect/pull/10477
+- Update kubernetes guide for readability — https://github.com/PrefectHQ/prefect/pull/10481
+- Fix a few typos in docs — https://github.com/PrefectHQ/prefect/pull/10488
+- Surface more documentation on push work pools — https://github.com/PrefectHQ/prefect/pull/10497
+- Fix heading size, remove unnecessary link in deployments documentation — https://github.com/PrefectHQ/prefect/pull/10489
+- Create GCP specific guide references for deploying a GKE cluster to host a worker — https://github.com/PrefectHQ/prefect/pull/10490
+- fix typo in gcs recipe — https://github.com/PrefectHQ/prefect/pull/10442
```
Comment (serinamarie) [inline RELEASE-NOTES.md:31]:
```suggestion
- Fix typo in `prefect-gcs` deployment example — https://github.com/PrefectHQ/prefect/pull/10442
```

File: RELEASE-NOTES.md [hunk 14]
```diff
@@ -1,5 +1,47 @@
...
+- Update warning block to mention worker in title — https://github.com/PrefectHQ/prefect/pull/10464
+- Fix duplicate Amplitude deviceId — https://github.com/PrefectHQ/prefect/pull/10449
+- Combine K8s worker page with K8s flows page — https://github.com/PrefectHQ/prefect/pull/10448
+- Guide - How to move data to and from cloud providers — https://github.com/PrefectHQ/prefect/pull/10133
+- Remove references to `flow_name` from deployments documentation — https://github.com/PrefectHQ/prefect/pull/10477
+- Update kubernetes guide for readability — https://github.com/PrefectHQ/prefect/pull/10481
+- Fix a few typos in docs — https://github.com/PrefectHQ/prefect/pull/10488
+- Surface more documentation on push work pools — https://github.com/PrefectHQ/prefect/pull/10497
+- Fix heading size, remove unnecessary link in deployments documentation — https://github.com/PrefectHQ/prefect/pull/10489
+- Create GCP specific guide references for deploying a GKE cluster to host a worker — https://github.com/PrefectHQ/prefect/pull/10490
```
Comment (serinamarie) [inline RELEASE-NOTES.md:30]:
```suggestion
- Add GCP-specific guide references for deploying a GKE cluster to host a worker — https://github.com/PrefectHQ/prefect/pull/10490
```

File: RELEASE-NOTES.md [hunk 15]
```diff
@@ -1,5 +1,47 @@
...
+- fix typo in gcs recipe — https://github.com/PrefectHQ/prefect/pull/10442
+- Move and reference Agents -> Workers upgrade guide — https://github.com/PrefectHQ/prefect/pull/10445
+Fix grammatical errors in documentation — https://github.com/PrefectHQ/prefect/pull/10457
```
Comment (serinamarie) [inline RELEASE-NOTES.md:33]:
```suggestion
- Fix grammatical errors in documentation — https://github.com/PrefectHQ/prefect/pull/10457
```

File: RELEASE-NOTES.md [hunk 16]
```diff
@@ -1,5 +1,61 @@
...
+We're happy to announce two new guides aimed at helping users get the most out of their Prefect setup!
+
+#### How to move data to and from cloud providers
+
+Moving data to cloud-based storage and retrieving it is crucial in many data engineering setups. This guide provides step-by-step instructions to seamlessly integrate and interact with popular cloud services like AWS, Azure, and GCP using Prefect.
+
+#### Running flows with Kubernetes
+
+For those aiming to optimize their flows using Kubernetes, this guide provides a deep dive on how to efficiently run flows on Kubernetes using containers. Catering to both novices and seasoned experts, the content offers insights for all proficiency levels.
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:16]:
```suggestion
```

File: RELEASE-NOTES.md [hunk 17]
```diff
@@ -1,5 +1,61 @@
...
+We're happy to announce two new guides aimed at helping users get the most out of their Prefect setup!
+
+#### How to move data to and from cloud providers
+
+Moving data to cloud-based storage and retrieving it is crucial in many data engineering setups. This guide provides step-by-step instructions to seamlessly integrate and interact with popular cloud services like AWS, Azure, and GCP using Prefect.
+
+#### Running flows with Kubernetes
+
+For those aiming to optimize their flows using Kubernetes, this guide provides a deep dive on how to efficiently run flows on Kubernetes using containers. Catering to both novices and seasoned experts, the content offers insights for all proficiency levels.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:15]: It might be nice to link to these guides in the docs

File: RELEASE-NOTES.md [hunk 18]
```diff
@@ -1,5 +1,60 @@
 # Prefect Release Notes
 
+## Release 2.11.5
+
+### New Guides
+
+We're happy to announce two new guides aimed at helping users get the most out of their Prefect setup!
```
Comment (billpalombi) [inline RELEASE-NOTES.md:7]:
```suggestion
We're happy to announce two new guides to help you get the most out of Prefect!
```

File: RELEASE-NOTES.md (hunk 17 above)
Comment (desertaxle) [reply RELEASE-NOTES.md:15]: Do we know what the URL will be since they haven't been published yet?

File: RELEASE-NOTES.md [hunk 19]
```diff
@@ -1,5 +1,60 @@
...
+
+#### How to move data to and from cloud providers
+
+Moving data to cloud-based storage and retrieving it is crucial in many data engineering setups. This guide provides step-by-step instructions to seamlessly integrate and interact with popular cloud services like AWS, Azure, and GCP using Prefect.
```
Comment (billpalombi) [inline RELEASE-NOTES.md:11]:
```suggestion
Moving data to cloud-based storage and retrieving it is crucial in many data engineering setups. This guide provides step-by-step instructions to seamlessly integrate and interact with popular cloud services like AWS, Azure, and GCP.
```

File: RELEASE-NOTES.md (hunk 17 above)
Comment (billpalombi) [reply RELEASE-NOTES.md:15]: +1

File: RELEASE-NOTES.md [hunk 20]
```diff
@@ -1,5 +1,60 @@
...
 
+## Release 2.11.5
+
+### New Guides
+
+We're happy to announce two new guides aimed at helping users get the most out of their Prefect setup!
+
+#### How to move data to and from cloud providers
+
+Moving data to cloud-based storage and retrieving it is crucial in many data engineering setups. This guide provides step-by-step instructions to seamlessly integrate and interact with popular cloud services like AWS, Azure, and GCP.
```
Comment (billpalombi) [inline RELEASE-NOTES.md:11]:
```suggestion
Moving data to cloud-based storage and retrieving it is crucial in many data engineering setups. [This guide](https://docs.prefect.io/latest/guides/moving-data) provides step-by-step instructions to seamlessly integrate and interact with popular cloud services like AWS, Azure, and GCP.
```

File: RELEASE-NOTES.md [hunk 21]
```diff
@@ -1,5 +1,60 @@
...
+
+We're happy to announce two new guides to help you get the most out of Prefect!
+
+#### How to move data to and from cloud providers
+
+Moving data to cloud-based storage and retrieving it is crucial in many data engineering setups. This guide provides step-by-step instructions to seamlessly integrate and interact with popular cloud services like AWS, Azure, and GCP.
+
+#### Running flows with Kubernetes
+
+For those aiming to optimize their flows using Kubernetes, this guide provides a deep dive on how to efficiently run flows on Kubernetes using containers. Catering to both novices and seasoned experts, the content offers insights for all proficiency levels.
```
Comment (billpalombi) [inline RELEASE-NOTES.md:15]:
```suggestion
For those aiming to optimize their flows using Kubernetes, this guide provides a deep dive on how to efficiently run flows on Kubernetes using containers. Catering to both novices and seasoned experts, [this guide](https://docs.prefect.io/latest/guides/deployment/kubernetes/) offers insights for all proficiency levels.
```

File: RELEASE-NOTES.md (hunk 17 above)
Comment (billpalombi) [reply RELEASE-NOTES.md]: Yes, I proposed revisions to include them

**Review by billpalombi — APPROVED** [review]
lgtm

**Review by serinamarie — APPROVED** [review]
LGTM :)

**Pull request merged**